tools/runners/test262-harness.py

```diff
@@ -392,16 +392,16 @@ def open_file(self):
         (self.file_desc, self.name) = tempfile.mkstemp(
             suffix=self.suffix,
             prefix=self.prefix,
-            text=self.text)
+            text=False)
 
     def write(self, string):
-        os.write(self.file_desc, string)
+        os.write(self.file_desc, string.encode("utf8"))
 
     def read(self):
-        file_desc = file(self.name)
+        file_desc = open(self.name, "rb")
         result = file_desc.read()
         file_desc.close()
-        return result
+        return result.decode("utf8", 'ignore')
```

Comment:
Don't mix single and double quotes. Preferably, use consistent style throughout the file. But at the very least, don't mix them in the same line.

Reply:
Some `decode`s use ignore, some don't. Why?

Reply:
only raised error on my gbk environment use ignore

Reply:
> Some `decode`s use ignore, some don't. Why?

done, all to be consistence